ceph_salt/config_shell.py

```diff
@@ -315,6 +315,12 @@ def possible_values(self):
                     },
                 }
             },
+            'Registries': {
+                'type': 'list',
+                'default': [],
+                'help': "List of custom registries in V2 format",
```

Comment:
how do I specify that this registry should be insecure? Should we add this to the help text?

Reply:
good point. It's a list of dicts with specific key names. I wonder how we could document that in the help text. By example maybe?

Reply:
Adding a sample in help text.